controller/src/command.rs

```diff
@@ -415,6 +426,8 @@ where
 	if method == "emoji" {
 		println!("\n\nThis is your response emoji string. Please send it back to the payer to finalize the transaction:\n\n{}", EmojiSlate().encode(&slate, receive_compressed));
 		info!("Response emoji.response generated, and can be sent back to the transaction originator.");
+	} else if method == "qr" {
+		QrToSlate((format!("{}{}", RESPONSE_EXTENTION, args.input)).into()).put_tx(&slate)?;
 	} else {
 		PathToSlate(format!("{}.response", args.input).into()).put_tx(&slate)?;
 		info!(
```

Comment:
A match in here too would be nice, even better with method being a enum

Reply:
I don't know if it needs to be an Enum, the QR method was based on the File method codes, but the writing functions are different.

The reading function can even be generalized.